src/command_modules/azure-cli-servicebus/HISTORY.rst

```diff
@@ -0,0 +1,10 @@
+.. :changelog:
+
+Release History
+===============
+
+0.0.1
```

Comment:
Change version no. to 0.1.0 for the initial release to properly follow SemVer.

Reply:
changed to 0.1.0